File: ts/src/BluefinShieldconexMgmtError.ts

import { Context } from './Context'


class BluefinShieldconexMgmtError extends Error {

  isBluefinShieldconexMgmtError = true

  sdk = 'BluefinShieldconexMgmt'

  code: string
  ctx: Context

  constructor(code: string, msg: string, ctx: Context) {
    super(msg)
    this.code = code
    this.ctx = ctx
  }

}

export {
  BluefinShieldconexMgmtError
}

File: dart/lib/BluefinShieldconexMgmtError.dart
class BluefinShieldconexMgmtError extends Error {
  final bool isBluefinShieldconexMgmtError = true;

  final String sdk = 'BluefinShieldconexMgmt';

  String code;
  String message;
  dynamic ctx;

  // Populated by makeError with the (cleaned) result and spec.
  dynamic result;
  dynamic spec;

  BluefinShieldconexMgmtError(this.code, this.message, [this.ctx]);

  @override
  String toString() => 'BluefinShieldconexMgmtError: ' + code + ': ' + message;
}
